pkg/pget.go

```diff
@@ -144,17 +148,24 @@ func (g *Getter) downloadAndMeasure(ctx context.Context, url, dest string, total
 	return nil
 }
 
-func (g *Getter) sendMetrics(url string, size int64) {
+func (g *Getter) sendMetrics(url string, size int64, throughput float64, downloadFailed bool) {
 	logger := logging.GetLogger()
 	endpoint := viper.GetString(config.OptMetricsEndpoint)
 	if endpoint == "" {
 		return
 	}
 
+	data := map[string]any{"url": url, "size": size, "version": version.GetVersion()}
+	if downloadFailed {
+		data["error"] = true
+	} else {
+		data["throughput"] = throughput
```

Comment:
Can we use something like `bytes_per_second` to be clearer and consistent with existing graph & https://github.com/replicate/hf_transfer/pull/1?